### ISSUE TITLE
"UnboundError" with backslash-command without argument (\c)

### ISSUE THREAD

**Issue by narzeja** [amjith/pgcli#28]
I get an uncaught exception with backslash-commands that take arguments, like \c.

Consider the following interaction with a freshly installed database.
```bash
$ pgcli testdb
testdb> \c testdb
You are now connected to database "testdb" as user "<my_user>"
testdb>
```

From here, if I type \c without specifying which database I wish to connect to, I get an uncaught exception:
```bash
testdb> \c
```
```python
Traceback (most recent call last):
  File "/home/<my_user>/.local/lib/python3.4/site-packages/pgcli/pgexecute.py", line 89, in run
    dbname = sql.split()[1]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/<my_user>/.local/lib/python3.4/site-packages/pgcli/main.py", line 104, in cli
    res = pgexecute.run(document.text)
  File "/home/<my_user>/.local/lib/python3.4/site-packages/pgcli/pgexecute.py", line 91, in run
    _logger.debug('Failed to switch. DB missing: %r', dbname)
UnboundLocalError: local variable 'dbname' referenced before assignment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/<my_user>/.local/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.10.0', 'console_scripts', 'pgcli')()
  File "/home/<my_user>/.local/lib/python3.4/site-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/home/<my_user>/.local/lib/python3.4/site-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/home/<my_user>/.local/lib/python3.4/site-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/<my_user>/.local/lib/python3.4/site-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/home/<my_user>/.local/lib/python3.4/site-packages/pgcli/main.py", line 116, in cli
    _logger.debug("sql: %r, error: %r", document.text, e.message)
AttributeError: 'UnboundLocalError' object has no attribute 'message'
```

A proper way of handling this is to inform the user that a database (or other argument for other backslash-commands) must be provided and return to the prompt gracefully.

System details:
OS: 3.17.6-1-ARCH
Python 3.4.2
postgres (PostgreSQL) 9.3.5
Latest pip-release of pgcli.

**Comment by amjith** [comment]
You're right. My bad. I'll fix this issue.

**Comment by narzeja** [comment]
Gazing over the code, the fix (of an uncaught exception while handling an exception) should be easy:
Change line 91 of pgexecute.py from:
```python
 _logger.debug('Failed to switch. DB missing: %r', dbname)
```
to
```python
_logger.debug('Failed to switch. No DB specified.')
```

Maybe?

**Comment by amjith** [comment]
This issue is now fixed by PR #49, #50 and commit: eed1931b3f77d5f9b4553ea41ebeeb3188d2a769

**Comment by amjith** [comment]
Version 0.10.1 released to PyPI. https://pypi.python.org/pypi/pgcli/0.10.1
